gui/locales/en.json

```diff
@@ -169,7 +169,8 @@
   "Unlinked Your device is no longer registered on your Cozy.": "Your device is no longer registered on your Cozy.",
   "Update": "Update",
   "Updater Checking for Update": "Checking for update…",
-  "Updater Downloading": "An update is available",
+  "Updater Downloading": "Updating Cozy Drive",
+  "Updater Downloading Second Line": "",
```

Comment:
What is the purpose of this locale key?

Reply:
Arf, I thought I'd removed it. I'll check this again.